fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidConstants.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.transfer.v1.fluid;
+
+public final class FluidConstants {
+	public static final long BUCKET = 81000;
+	public static final long BOTTLE = 27000;
+	public static final long INGOT = 9000;
+	public static final long NUGGET = 1000;
```

Comment:
I think it would be better to define these off of each other, that way if we ever change the amount of one, it does it for all
```suggestion
	public static final long BUCKET = 81000;
	public static final long BOTTLE = BUCKET / 3;
	public static final long INGOT = BUCKET / 9;
	public static final long NUGGET = INGOT / 9;
```

Reply:
*Also make 81000 not a compile constant (`long BUCKET = Objects.requireNonNull(81000);`) so I can break it with a mod*

Reply:
These constants must never be changed in the future, for obvious reasons... If they need to be updated, we will need to deprecate the older classes, and create new ones instead.

Reply:
Defining them like how @OroArmor suggested is easier to understand imo. And if required, why not change these values within snapshot cycles instead of deprecating the class? Since they're all compile time constants, deleting the class wouldn't break old mods anyway

Reply:
No, there is nothing worse than silently changing values, be it during snapshot cycles or not. You can't possibly know that your code still compiles but is now broken after a FAPI update.

I prefer using the direct values, as they are more explicit. If the API user can't understand why an ingot is 9000 droplets after I add javadoc I don't think we can expect them to understand the rest of the API anyway.